content/guidelines/user-assistance/ua guidelines.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Links on this page pointing elsewhere within the documentation should be relative.